src/com/hillel/course2/lesson10/homework/Main.java

```diff
@@ -0,0 +1,36 @@
+package com.hillel.course2.lesson10.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+        Main.Counter counter = new Main.Counter();
+            List<Thread> threads = new ArrayList<>();
+            for (int i = 0; i < 10; i++) {
+                Thread thread = new Thread(counter);
+                threads.add(thread);
+                thread.start();
+
+            }
+        for (Thread thread: threads) {
+            thread.join();
+        }
+        System.out.println(Counter.COUNTER);
```

Comment:
What do you see in the output? Run 5-10-20 times. Is the result always the same or not?